webpack.common.ts

```diff
@@ -37,18 +48,14 @@ module.exports = {
         'node_modules/monaco-languageclient/lib/vscode-compatibility'
       ),
     },
-    extensions: ['.tsx', '.ts', '.js', '.wasm'],
+    extensions: ['.tsx', '.ts', '.js', '.wasm', '...'],
+    fallback: {
+      path: false,
+    },
   },
+  ignoreWarnings: [/export .* was not found in/, /'.\/locale' in/],
   node: {
-    fs: 'empty',
     global: true,
-    crypto: 'empty',
-    tls: 'empty',
-    net: 'empty',
-    process: true,
-    module: false,
-    clearImmediate: false,
-    setImmediate: true,
   },
```

Comment:
What's the rationale for removing these properties from the node object?

Reply:
@wdoconnell these existing failed the build because the `node` object no longer accepts these keys except for `Buffer`, `process`, (which is false by default), and `global`. Webpack 5 changed how the default polyfills it includes work. This may be helpful: https://gist.github.com/ef4/d2cf5672a93cf241fd47c020b9b3066a